.circleci/config.yml

```diff
@@ -360,6 +360,7 @@ jobs:
       - store-artifacts-and-test-results
       - run:
           name: Notify ICFY
+          no_output_timeout: 20m
```

Comment:
Why is the timeout on the "notify" task rather than the "build stats" one? Shouldn't it be the other way around?